### DISCUSSION TITLE
How can I create an annotated argument without a default value?

### DISCUSSION THREAD

**Discussion by justinclloyd** [fastapi/typer#616] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the Typer documentation, with the integrated search.
- [X] I already searched in Google "How to X in Typer" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to Typer but to [Click](https://github.com/pallets/click).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
# First example on https://typer.tiangolo.com/tutorial/options/required/
import typer
from typing_extensions import Annotated

def main(name: str, lastname: Annotated[str, typer.Option()]):
    print(f"Hello {name} {lastname}")

if __name__ == "__main__":
    typer.run(main)
```

### Description

The example code doesn't work, throwing the following exception saying that Option() needs a default argument.
```
Traceback (most recent call last):
  File "./t.py", line 6, in <module>
    def main(name: str, lastname: Annotated[str, typer.Option()]):
TypeError: Option() missing 1 required positional argument: 'default'
```

Ultimately, I'm trying to create a Typer command that takes a required pathlib.Path argument for which there is no sane default value. I could make one up, even use a fake location, but I feel this should be possible.

### Operating System

Linux

### Operating System Details

Ubuntu 20.04

### Typer Version

0.4.2

### Python Version

3.8.10

**Comment by pmav99** [comment, marked as answer]
1. You wrote that you use version 0.4.2. AFAIK version 0.4.2 doesn't support `Annotated`, yet you ask for `Annotated`. If you want to use `Annotated` I would suggest that you upgrade to 0.9.0+
3. To answer the question of the title, an example of `Annotated` argument with a default value (i.e. an *optional argument*) can be found in the [docs](https://typer.tiangolo.com/tutorial/arguments/optional/#make-an-optional-cli-argument)
4. You wrote: 
> a Typer command that takes a required pathlib.Path argument for which there is no sane default value.

Probably some terminology miscommunication here, but If the argument has a default value, then it is an optional argument, is it not?
